### PR TITLE
Add course: Python for OSINT. 21 day course for beginners (#9362)

### PR DESCRIPTION
**Pull request by KairoQalb**
## What does this PR do?
Add resource(s) 

## For resources
Python
### Description
This pull request is solving issue #9362 

### Why is this valuable (or not)?
This is a course for beginners who want to learn Python for OSINT. It's beginner-friendly with full of details and valuable insights. Anyone can download the PDF for free and without any signup. 

### How do we know it's really free?
It's an open-source course repository on GitHub with a free PDF.

### For book lists, is it a book? For course lists, is it a course? etc.
It's a course.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by KairoQalb** [comment]
Sorry, I think I made a mistake in alphabetical ordering.

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -1216,6 +1216,7 @@
 * [Python Course from scratch](https://scrimba.com/playlist/pNpZMAB) - Olaf Paulson (scrimba)
 * [Python Data Analysis](https://www.coursera.org/learn/python-analysis) - Scott Rixner, Joe Warren (Coursera)
 * [Python for Beginners (Full Course)](https://www.youtube.com/playlist?list=PLsyeobzWxl7poL9JTVyndKe62ieoN-MZ3) - Telusko
+* [Python for OSINT. 21 day course for beginners](https://github.com/cipher387/python-for-OSINT-21-days) - cyb_detective (GitHub Repository with PDF)
```
Comment (eshellman) [inline courses/free-courses-en.md:1219]:
```suggestion
* [Python for OSINT. 21 day course for beginners](https://github.com/cipher387/python-for-OSINT-21-days/raw/main/Python%20for%20OSINT.%2021%20day%20course%20for%20beginners.pdf) - cyb_detective (PDF)
```

**Review by eshellman — CHANGES_REQUESTED** [review]
Be sure to check the free-programming-books-lint output for linter errors. Alphabetical order and spacing are the most common problems.

**Comment by KairoQalb** [comment]
> Be sure to check the free-programming-books-lint output for linter errors. Alphabetical order and spacing are the most common problems.

First, the error indicated swapping l.1220 and l.1219  alphabetize-lists  remark-lint 
Now it's: swap l.1221 and l.1220  alphabetize-lists  remark-lint
I'm sorry for these errors.
Should I close this pull request?

**Comment by KairoQalb** [comment]
> > First, the error indicated swapping l.1220 and l.1219 alphabetize-lists remark-lint Now it's: swap l.1221 and l.1220 alphabetize-lists remark-lint I'm sorry for these errors. Should I close this pull request?
> 
> No no @KairoQalb. Just make the stated changes You are good to go! 👍

Thanks! Okay, I get it. All checks are cleared now.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**